packages/front-end/components/Dimensions/DimensionForm.tsx

```diff
@@ -73,8 +73,9 @@ const DimensionForm: FC<{
         placeholder="Choose one..."
         options={datasources.map((d) => ({
           value: d.id,
-          label: d.name,
+          label: `d.name${d.description ? ` — ${d.description}` : ""}`,
```

Comment:
Did you mean to interpolate `d.name` to? like:

```
`${d.name}${d.description ? ` — ${d.description}` : ""}`
```

The code looks like it would print out `d.name`. Has this one been captured in the screenshots?

Reply:
I just tested it out, the issue is as I suspected:

```js
> var d = { name: 'My Postgres DB', description: 'This is my Postgres DB!' }
undefined
> `d.name${d.description ? ` — ${d.description}` : ""}`
'd.name — This is my Postgres DB!'
```

The screenshot looks fine though. 🤔

Reply:
Yikes, I think something got messed up when resolving a merge conflict recently. I’ll fix this shortly